kubernetes/multi-tenant/README.md

```diff
@@ -0,0 +1,6 @@
+## Script defaults
+
+* Creates a 10 MB persistent volume to load the multi tenant config file
+* Creates a 2 GB persistent volume recordings to store recordings
+* Recording is disabled by default. To enable recording set the value of ENABLE_RECORDING variable to true
+* Need to add credentials to tenantConfig.json and copy it to /root/tenantConfig folder on worker node
```

Comment:
So does this mean you have to copy the multi-tenant file to each node in the cluster or can all the worker nodes use the same volume?